roles/ceph-mon/tasks/deploy_monitors.yml

```diff
@@ -0,0 +1,42 @@
+---
+- name: generate monitor initial keyring
+  local_action: >
+    shell python -c "import os ; import struct ; import time; import base64 ; key = os.urandom(16) ; header = struct.pack('<hiih',1,int(time.time()),0,len(key)) ; print base64.b64encode(header + key)" | tee fetch/monitor_keyring.conf
+    creates=fetch/monitor_keyring.conf
+  register: monitor_keyring
+  sudo: false
+  when: monitor_secret != 'AQAWqilTCDh7CBAAawXt6kyTgLFCxSvJhTEmuw=='
```

Comment:
Why does this need to write it out on the local filesystem if the output is registered? Also, this should be a script file rather than raw python in the task parameters.